two curl failures in the mono lanes 

https://dev.azure.com/dnceng-public/public/_build/results?buildId=461322&view=logs&j=7f49df26-8126-5de3-bf2f-6ac6bde01830&t=af2c051c-0c65-59f6-e97b-23ca21856a55&l=15

```
/bin/bash --noprofile --norc /Users/runner/work/_temp/b2a99c01-a851-429e-974f-76d8d10baf16.sh
Downloading 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh'
curl: (35) Send failure: Broken pipe
Curl failed; dumping some information about dotnet.microsoft.com for later investigation
write:errno=54
CONNECTED(00000006)
---
no peer certificate available
---
No client certificate CA names sent
---
SSL handshake has read 0 bytes and written 322 bytes
Verification: OK
---
New, (NONE), Cipher is (NONE)
Secure Renegotiation IS NOT supported
Compression: NONE
Expansion: NONE
No ALPN negotiated
Early data was not sent
Verify return code: 0 (ok)
---
##[error]Bash exited with code '1'
```

--- a/src/tests/Loader/classloader/regressions/GitHub_93770/GitHub_93770.cs
+++ b/src/tests/Loader/classloader/regressions/GitHub_93770/GitHub_93770.cs
@@ -1,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace ReproGH93770;
+
+// By default in Mono every class that implements interfaces has 19
+// interface method table slots that are used to dispatch interface
+// calls.  The methods are assigned to slots based on a hash of some
+// metadata.  This test tries to create at least one IMT slot that has
+// a collision by creating an interface with 20 virtual methods.
+//
+// The bug is that if the method also has a non-virtual generic
+// method, the IMT slot with the collision calls the wrong vtable
+// element.
+
+public class ReproGH93770
+{
+    [Fact]
+    public static int TestEntryPoint()
+    {
+        C c = new C();
+        I1 i1 = c;
+        Helper(i1);
+        return 100;
+    }
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static void Helper(I1 i1)
+    {
+    var e = 0;
+    var n = i1.M0();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M1();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M2();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M3();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M4();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M5();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M6();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M7();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M8();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M9();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M10();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M11();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M12();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M13();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M14();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M15();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M16();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M17();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M18();
+    Assert.Equal(e, n);
+    e++;
+    n = i1.M19();
+    Assert.Equal(e, n);
+    e++;
+    }
+}
+
+public interface I1 {
+    public static T Id<T> (T t)=> t;
+
+    public int M0() => 0;
+    public int M1() => 1;
+    public int M2() => 2;
+    public int M3() => 3;
+    public int M4() => 4;
+
+    public int M5() => 5;
+    public int M6() => 6;
+    public int M7() => 7;
+    public int M8() => 8;
+    public int M9() => 9;
+    public int M10() => 10;
+    public int M11() => 11;
+    public int M12() => 12;
+    public int M13() => 13;
+    public int M14() => 14;
+    public int M15() => 15;
+    public int M16() => 16;
+    public int M17() => 17;
+    public int M18() => 18;
+    public int M19() => 19;
+
+}
+
+public class C : I1 {
+
+}
+
+

--- a/src/tests/Loader/classloader/regressions/GitHub_93770/GitHub_93770.csproj
+++ b/src/tests/Loader/classloader/regressions/GitHub_93770/GitHub_93770.csproj
@@ -1,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="GitHub_93770.cs" />
+  </ItemGroup>
+</Project>

